### PR TITLE
Create dgbcombine.py

### PR DESCRIPTION
**Pull request by hpratama**

### CODE REVIEW

File: dgbpy/dgbcombine.py [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+from skl2onnx.common.data_types import FloatTensorType
+import sclblonnx as so
+
+
+savetypes = ['onnx', 'joblib', 'pickle', 'xgboost']
+modeltypes = ['classification', 'regression']
+defsavetype = savetypes[2]
+defmodeltype = modeltypes[0]
+
+def fit(models, inputs, outputs): # models, Xs, ys
```
Comment (arnauddgbes) [inline dgbpy/dgbcombine.py]: Maybe nice to have, but does not fit as such in our workflow. You may copy it for reference to dgbscikit.py

File: dgbpy/dgbcombine.py [hunk 2]
```diff
@@ -0,0 +1,131 @@
...
+    elif model_type == modeltypes[1]:
+        input_names = [i.name for i in sess.get_inputs()]
+        output_names = [o.name for o in sess.get_outputs()]
+        input_feed = {}
+        for i in input_names:
+            input_feed[i] = X.astype(np.float32)
+        outputs = sess.run(output_names, input_feed)
+        return np.mean(np.column_stack(outputs), axis=1)   
+
+def save_model(models, outfnm, save_type=defsavetype, input_shape=None):
```
Comment (arnauddgbes) [inline dgbpy/dgbcombine.py]: Mostly duplicates dgbscikit.py:save
Let me propose the following signature:
dgbscikit.py:save( model, outfnm, save_type=defsavetype, prior_models=None )

Where prior_models is a set of already trained models.
If provided, you can add 'model' to the list 'prior_models', and save that extended list

File: dgbpy/dgbcombine.py [hunk 3]
```diff
@@ -0,0 +1,131 @@
...
+            combined_model.append(so.concat(combined_model[i-2], sgs[i].graph, rename_nodes=True, rename_edges=False, rename_io=False))
+        so.graph_to_file(combined_model[-1], outfnm)
+    elif save_type == savetypes[1]:
+        return joblib.dump(models, open(outfnm, 'wb'), compress=1)
+    elif save_type == savetypes[2]:
+        return pickle.dump(models, open(outfnm, 'wb'))
+    else:
+        return pickle.dump(models, open(outfnm, 'wb'))
+
+def load_model(modelfnm, save_type=defsavetype):
```
Comment (arnauddgbes) [inline dgbpy/dgbcombine.py]: I am not sure if this function does anything different from dgbpy.dgbscikit.load, so it can probably be removed.

**Review by arnauddgbes — CHANGES_REQUESTED** [review]

**Comment by marxprop** [comment]
@hpratama , I have compiled a detailed gist along with codes on the first steps which include setting up the train function in `dgbpy.dgbscikit` to handle training with multiple chunks and multiple folds. You can get ideas from this to help with your implementation. 

You can find it - [here](https://gist.github.com/marxprop/e321b7544a09d028cb4a7b21ebe4059d)

**Comment by arnauddgbes** [comment]
Please rebase, and I can do a QC again

**Review by arnauddgbes — APPROVED** [review]

**Pull request closed without merging**